metabase/metabase
# Allow dropdown source for all string operators in dashboards

How to verify:
- Add string/contains parameter to the dashboard and connect it to a field
- Set "dropdown" options in the sidebar for this parameter
- Make sure you can select field values in the widget for this parameter
- Custom value source options should also work

<img width="1603" alt="Screenshot 2024-03-06 at 18 38 27" src="https://github.com/metabase/metabase/assets/8542534/192046fc-aa33-48c2-aa2d-afbcde611685">
<img width="653" alt="Screenshot 2024-03-06 at 18 38 33" src="https://github.com/metabase/metabase/assets/8542534/f2d5f5d1-764f-4886-a21f-ae7714e15631">
